Location.md

```diff
@@ -20,6 +20,32 @@ See HTML.
 
 See HTML.
 
+## Location.toString ( )
+
+1. Let _hrefDesc_ be DefaultInternalMethod([[GetOwnProperty]\], this, "href").
+
+1. Return the result of calling _hrefDesc_.[[Get]\].
+
+The property has the attributes { [[Writable]]: false, [[Enumerable]]: true, [[Configurable]]: false }.
+
+## Location.toJSON
+
+The initial value of Location.toJSON is undefined.
```

Comment:
It needs to be specced non-writable non-configurable, value undefined, so pages can't mess with it.  Assuming we don't want pages spoofing JSON.stringify(location), at least, which is the current intent of IDL.